src/com/redhat/ThreescaleService.groovy

```diff
@@ -2,72 +2,91 @@
 
 package com.redhat
 
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurperClassic
```

Comment:
If my memory serves, those classes trigger a sandbox exception in Jenkins that needs to be manually approved. To make the pipeline usable without any jenkins configuration (and thus apply "configuration-as-code" principles), I usually try to avoid them and use the `readJSON` step instead.

Reply:
Agreed and used readJSON function